cypress/support/step-definitions/common-steps.js

```diff
@@ -230,6 +265,21 @@ Then('labelAlign on preview is {string}', (direction) => {
   }
 });
 
+Then('{string} label Align on preview is {string}', (position, direction) => {
+  switch (position) {
+    case 'First':
+      labelByPosition(FIRST_ELEMENT).should('have.css', TEXT_ALIGN, `${direction}`);
```

Comment:
no need to pass the parameter using ${parameter} if we are not combining strings, just pass `direction`

Reply:
fixed